lib/resty/radixtree.lua

```diff
@@ -114,10 +114,12 @@ ffi_cdef[[
     void *radix_tree_find(void *t, const unsigned char *buf, size_t len);
     void *radix_tree_search(void *t, void *it, const unsigned char *buf,
         size_t len);
-    int radix_tree_pcre(void *it, const unsigned char *buf, size_t len);
+    int radix_tree_pre(void *it, const unsigned char *buf, size_t len);
```

Comment:
I think the name is radix_tree_prev according to the name conversion.

Reply:
ok